### ISSUE TITLE
Missing required module 'Stripe3DS2'

### ISSUE THREAD

**Issue by dave-perry** [stripe/stripe-ios#1750]
## Summary
When I try and build the app using version 21.2.0 of the SDK I get an error of `Missing required module 'Stripe3DS2'`

## Code to reproduce
N/A

## iOS version
All

## Installation method
SPM

## SDK version
21.2.0

## Other information
We're attempting to migrate to using the Stripe framework via SPM instead of Cocoapods. Everything works fine for v21.1.0 but when we try to use v21.2.0 we get the above error.

Note we're integrating Stripe in an internal framework and then using that internal framework in the app.

**Comment by davidme-stripe** [comment]
Thanks for filing, I'll look into this. It's possible that SPM isn't reading the dependency chain correctly, so we might need to export Stripe3DS2 as a separate package that your intermediate framework can depend on alongside Stripe.

Just to make sure, have you tried a clean and rebuild and `File -> Swift Packages -> Reset Package Caches`? I encountered a similar error a few times when switching between 21.1.0 and 21.2.0, but it seemed to be an Xcode caching bug.

**Comment by dave-perry** [comment]
Thanks. I just tried your instructions and am still receiving the same error.

Your solution sounds promising - please let me know if I can aid in any way, or if you need something testing.

**Comment by davidme-stripe** [comment]
Okay, thanks for trying!

I've created a branch which exports Stripe3DS2: Try pointing SPM at the [`davidme/export-stripe3ds2`](https://github.com/stripe/stripe-ios/tree/davidme/export-stripe3ds2) branch, then importing both `Stripe3DS2` and `Stripe` within your library.

**Comment by dave-perry** [comment]
Thanks for the quick response.

I tried this and still had the same issue.

However, while further debugging I realised this was an error on our part - I'll put some detail below before closing this issue in case someone else comes across the same problem...

We have several internal frameworks in our project and in one of them contains our payment code, linking the Stripe framework. Let's call this the payment framework.

Our payment framework is also linked to another framework for our UI.

Both these frameworks are also linked to our main app. Somehow I had removed the linking of the UI framework and had I paid more attention I would have spotted that the error I mentioned in the initial post was being emitted by this UI framework.

Appreciate your help @davidme-stripe
